src/factory_generator/app/tests/assets/test_item_factory/subdir/factory3.cc

```diff
@@ -0,0 +1,22 @@
+should remain
```

Comment:
Quelle est l'utilité du "should remain"?

Reply:
Le filter generator devrait pas modifier cette ligne là. Elle devrait rester là (pour s'assurer que le generator efface pas autre chose)